Automattic/wp-calypso
# Jetpack section: Jetpack, Backup and Scan shouldn't be translated

#### Changes proposed in this Pull Request

On the sidebar, the menu: Jetpack, Backup and Scan shouldn't be translated. They are brand and product names.

More context: p1591780348010800-slack-jetpack-voyager

#### Testing instructions

- Apply this change
- Select a Jetpack site in Calypso
- Change your account language
- You should see the Jetpack, Backup and Scan menu names in English
